app.js

```diff
@@ -64,12 +64,12 @@ function save() {
 
 function load() {
   let parsedProducts = localStorage.getItem("productData");
-  JSON.parse(parsedProducts);
+  parsedProducts = JSON.parse(parsedProducts);
```

Comment:
After looking at this function, I recommend console logging `parsedProducts` to better understand what it is.